db/etl/import_us_cities.py

```
@@ -0,0 +1,224 @@
+#!/usr/bin/env python3
+"""
+Script to import US cities from the countries-states-cities-database
+into the PostgreSQL database using psycopg3.
+"""
+
+import os
+import re
```

Ruff (0.8.2)

`re` imported but unused

Remove unused import: `re`

(F401)